controllers/discordactions.js

```diff
@@ -383,6 +383,74 @@ const setRoleToUsersWith31DaysPlusOnboarding = async (req, res) => {
   }
 };
 
+const generateInviteForUser = async (req, res) => {
+  try {
+    const { userId } = req.query;
+    const userIdForInvite = userId || req.userData.id;
+
+    const modelResponse = await discordRolesModel.getUserDiscordInvite(userIdForInvite);
+
+    if (!modelResponse.notFound) {
+      return res.status(409).json({
+        message: "User invite is already present!",
+      });
+    }
+
+    const channelId = config.get("discordNewComersChannelId");
+    const authToken = jwt.sign({}, config.get("rdsServerlessBot.rdsServerLessPrivateKey"), {
+      algorithm: "RS256",
+      expiresIn: config.get("rdsServerlessBot.ttl"),
+    });
+
+    const inviteOptions = {
+      channelId: channelId,
+    };
+    const response = await fetch(`${DISCORD_BASE_URL}/invite`, {
+      method: "POST",
+      body: JSON.stringify(inviteOptions),
+      headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+    }).then((response) => response.json());
+
```

Comment:
NIT: this could have been in a util

Reply:
not actually this is going to be used only here, I don't think there is a need to put it in a util file

Reply:
okay